### PR TITLE
Display new plugins details

### PR DESCRIPTION
**Pull request by kingsamurai123**
Whenever new plugins get added to the manager, the game displays the count and names of the plugins. #64

### CODE REVIEW

**Comment by Loup-Garou911XD** [comment]
https://github.com/bombsquad-community/plugin-manager/blob/97a4a6a244e3bf78366c5f14600c1f2ac4dd8340/plugin_manager.py#L220
Installed plugins are stored in config

**Comment by kingsamurai123** [comment]
This does provide only installed ones. What about not installed plugins?
We need to check with them as well right? To get the new plugins. 
If not this will popup each time if any plugin is not installed.

**Comment by Loup-Garou911XD** [comment]
Oh okay I thought current plugins meant like the installed ones

**Comment by rikkolovescats** [comment]
I don't feel maintaining a list of not-installed plugins locally would be a good idea.

Another solution that should be good enough would be to check if the count doesn't match here:
https://github.com/bombsquad-community/plugin-manager/blob/97a4a6a244e3bf78366c5f14600c1f2ac4dd8340/plugin_manager.py#L286

If it doesn't then can sort all downloadable plugins having only one release version by their `released_on` date in descending order:
https://github.com/bombsquad-community/plugin-manager/blob/97a4a6a244e3bf78366c5f14600c1f2ac4dd8340/plugins/utilities.json#L25

We can then caclulate the difference between the new available count and the locally stored count and figure out newly available plugins from the previously sorted plugins list.

**Comment by kingsamurai123** [comment]
Your point does give a new solution to the problem. Thank you Rikko.

**Comment by kingsamurai123** [comment]
Changed the logic, but ended up adding a new property in Plugin class along with importing datetime method.
If this is fine, will start 106 and can merge both at same time.

**Comment by kingsamurai123** [comment]
Don't merge this, I made mistake in taking release_date as key for the dictionary and just now remembered that, the release dates can be same for different plugins and this would give a key error. Making the changes to take plugin name as key and also checking API version.

File: plugin_manager.py [hunk 1]
```diff
@@ -268,6 +269,33 @@ async def update_plugins(self):
...
+            latest_version = max(value['versions'].keys())
+            new_plugins[key] = {'released_on': value['versions'][latest_version]['released_on'],
+                                'api_version': value['versions'][latest_version]['api_version']}
+            sorted_plugins = dict(sorted(new_plugins.items(),
+                                         key=lambda x: datetime.strptime(
+                                             x[1]['released_on'], '%d-%m-%Y'),
+                                         reverse=True))
+
+        for key, value in sorted_plugins.items():
+            if count < new_plugin_count:
```
Comment (kingsamurai123) [inline plugin_manager.py:290]: Need help in here to check the api_version. It is present in the dictionary values, but I am not sure how check that with the device.

**Comment by kingsamurai123** [comment]
Why the md5 checksum failed?

**Review by rikkolovescats — APPROVED** [review]
thank you so much again!!

**Comment by kingsamurai123** [comment]
Thank you rikko!

**Pull request merged**